s/actuator/actuator.ts

```diff
@@ -0,0 +1,104 @@
+
+import "@babylonjs/core/Collisions/collisionCoordinator.js"
+import "@babylonjs/core/Materials/standardMaterial.js"
+import "@babylonjs/core/Lights/Shadows/index.js"
+import "@babylonjs/loaders/glTF/2.0/index.js"
+import "@babylonjs/core/Culling/ray.js"
+
+import {Oracle} from "../oracle/oracle.js"
+import {makeGround} from "../landscape/ground.js"
+import {makeTheater} from "../theater/theater.js"
+import {makeSettings} from "../settings/settings.js"
+import {setupLighting} from "../landscape/lighting.js"
+import {makeRandomToolkit} from "../toolbox/randomly.js"
+import {sprinkleProps} from "../landscape/sprinkle-props.js"
+import {makeSpectatorCamera} from "../cameras/spectator-camera.js"
+
+
+export function makeActuator({
+		oracle
+	}: {
+		oracle: Oracle
+	}) {
+
+	const settings = makeSettings()
+	const theater = makeTheater()
+
+	function resizeAll() {
+		theater.onresize()
+	}
+	window.addEventListener("resize", resizeAll)
+	document.addEventListener("fullscreenchange", resizeAll)
+	setTimeout(resizeAll, 0)
+
+
+	return {
+		theater,
+		settings,
+		async initialize() {
+			const mapSize = 1000
+			const cliffSlopeFactor = 0.4
+			const randomly = makeRandomToolkit()
+
+			const {updateTargetHeight, smoothUpdateForCameraHeight} = makeSpectatorCamera({
+				theater,
+				sampleHeight: oracle.sampleHeight
+			})
+
+			let gravityEnabled = settings.readable.enableGravityAndCollisions
```

Comment:
this is where we toggle gravity and collision, also calling functions that apply them